### ISSUE TITLE
Classifiers developed on a different domain crash

### ISSUE THREAD

**Issue by ajdapretnar** [biolab/orange3#1763]
##### Orange version
3.3.9.dev0+909a119

##### Expected behavior
Predictions should raise an error message instead of crashing

##### Actual behavior
Predictions crashes when fed the wrong classifier

##### Steps to reproduce the behavior
File - Predictions
Load Classifier (classifier should be trained on a different data set)

##### Additional info (worksheets, data, screenshots, ...)
```
ValueError Exception
Traceback (most recent call last):
  File "c:\orange\orange3\Orange\canvas\scheme\widgetsscheme.py", line 832, in process_signals_for_widget
    widget.handleNewSignals()
  File "c:\orange\orange3\Orange\widgets\evaluate\owpredictions.py", line 268, in handleNewSignals
    self.commit()
  File "c:\orange\orange3\Orange\widgets\evaluate\owpredictions.py", line 457, in commit
    predictions.metas[:, -newcolumns.shape[1]:] = newcolumns
ValueError: could not broadcast input array from shape (150,4) into shape (150,3)
```

**Comment by zafar11235** [comment]
Is somebody working on this?

**Comment by ajdapretnar** [comment]
Not as far as I know.

**Comment by ajdapretnar** [comment]
Fixed via #2183.
